Apps/APN/src/services/CustomerIOService.js

```diff
@@ -0,0 +1,70 @@
+import {
+  CioLogLevel,
+  CustomerIO,
+  CustomerIOEnv,
+  CustomerioConfig,
+} from 'customerio-reactnative';
+
+class CustomerIOService {
```

Comment:
Is there a reason for abstracting the RN SDK into it's own class? 

I see more value in not abstracting the SDK calls into a class and instead call the SDK directly in the app's components. I believe this makes a better code sample for people browsing the sample app to learn how to use our SDK. 

If someone reads our docs for how to track an event and they see a code snippet `CustomerIO.track()` but then they see in the sample app `CustomerIOService.track()`, that is confusing.

Reply:
Thanks for sharing Levi. I agree that the `CustomerIOService.track()` method can be slightly confusing. However, the reason I decided to keep all SDK calls in a separate file is that for someone looking at the project for the very first time, it might not be easy to determine the exact location of the relevant call. For example, the identify call could be in a UI component, storage class, API call, repository, or elsewhere depending on the project structure.

By having a separate class, it becomes slightly easier to locate the relevant code. Regarding the naming, we can consider renaming it to minimize confusion. We can use `SDKService.onUserLoggedIn()`, `SDKService.onEventTracked()`, or even remove the class and use functions directly, such as `onUserLoggedIn()` and `onEventTracked()`?